src/librustc_resolve/lib.rs

```diff
@@ -1610,6 +1611,12 @@ impl<'a> Resolver<'a> {
         DefCollector::new(&mut definitions, Mark::root())
             .collect_root(crate_name, session.local_crate_disambiguator());
 
+        let mut extern_prelude: FxHashSet<Name> =
+            session.opts.externs.iter().map(|kv| Symbol::intern(kv.0)).collect();
+        for name in crates_to_inject(&krate) {
```

Comment:
I don't think we need to add `compiler_builtins` to the prelude though, like `crates_to_inject` does.

Reply:
Fixed.